components/org.wso2.carbon.identity.auth.otp.core/src/main/java/org/wso2/carbon/identity/auth/otp/core/util/AuthenticatorUtils.java

```diff
@@ -59,4 +63,25 @@ public static String maskIfRequired(String value) {
 
         return LoggerUtils.isLogMaskingEnable ? LoggerUtils.getMaskedContent(value) : value;
     }
+
+    public static Property[] getAccountLockConnectorConfigs(String tenantDomain) throws
+            AuthenticationFailedException {
+
+        Property[] connectorConfigs;
+        try {
+            connectorConfigs = AuthenticatorDataHolder
+                    .getIdentityGovernanceService()
+                    .getConfiguration(
+                            new String[]{
+                                    AuthenticatorConstants.PROPERTY_LOGIN_FAIL_TIMEOUT_RATIO,
+                                    AuthenticatorConstants.PROPERTY_ACCOUNT_LOCK_ON_FAILURE,
+                                    AuthenticatorConstants.PROPERTY_ACCOUNT_LOCK_ON_FAILURE_MAX,
+                                    AuthenticatorConstants.PROPERTY_ACCOUNT_LOCK_TIME
+                            }, tenantDomain);
+        } catch (Exception e) {
+            throw new AuthenticationFailedException("Error occurred while retrieving account lock connector " +
```

Comment:
Let's add the tenant domain to the error message

Reply:
fixed